packages/forms/src/UIForm/fieldsets/CollapsibleFieldset/CollapsibleFieldset.component.js

```diff
@@ -55,6 +55,10 @@ export function defaultTitle(formData, schema, options) {
 	if (title.length > 0) {
 		return title.join(options?.separator || schema.options?.separator || ', ');
 	}
+
+	if (options?.emptyTitleFallback) {
+		return options.emptyTitleFallback || schema.options?.emptyTitleFallback;
```

Comment:
i think `options.emptyTitleFallback` will always be return with the current condition. Not sure this is intended